platform/iphone/gamepad_iphone.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+@interface GamepadManager : NSObject
```

Comment:
Verrrry simple object for tracking active gamepads and routing their messages back to Godot.